cmd/install/assets/hypershift_operator.go

```diff
@@ -584,9 +598,10 @@ func (o HyperShiftOperatorDeployment) Build() *appsv1.Deployment {
 	}
 
 	if o.OpenShiftTrustBundle != nil {
-		deployment.Spec.Template.Spec.Containers[0].VolumeMounts = append(deployment.Spec.Template.Spec.Containers[0].VolumeMounts, corev1.VolumeMount{
+		optionalTrustBundle := true
```

Comment:
This is conditional `if o.OpenShiftTrustBundle != nil {`
But the install command is always enforcing its creation https://github.com/openshift/hypershift/blob/bb04b0a79621406a3c0b7522e50ee51cce70f85f/cmd/install/install.go#L483-L544

am I missing anything?

Reply:
We actually need to not enforce its creation. Always creating it and then mounting the volume here is causing issues for IBM.

Reply:
I was trying to fix that with Cesar in https://github.com/openshift/hypershift/pull/2744 but that PR is not working. Cesar thought we would need to wait for this PR, or I suppose we can try to fix it within this PR.

Reply:
With what Bryan has mentioned, I'll leave this conditional here since the `OpenshiftTrustBundle` may not exist in the future. Please let me know or reopen this conversation if that is not the case.